package.nls.json

```diff
@@ -47,6 +51,10 @@
     "cmake-tools.command.cmake.cleanRebuildAll.title": "Clean Rebuild All Projects",
     "cmake-tools.command.cmake.ctest.title": "Run Tests",
     "cmake-tools.command.cmake.ctestAll.title": "Run Tests for All Projects",
+    "cmake-tools.command.cmake.cpack.title": "Run CPack",
```

Comment:
When playing with the extension, I wondered if we wanted to modify this name to include the fact that this is running cpack for a package preset? 

Currently I don't think there is support for just generically running cpack, similar to how there isn't support for just running cmake generically, so I would imagine it could be useful to have something similar to "Package Preset" or "Run Package Preset" or "Run CPack for Package Preset", something of the sort.

Reply:
Ignore this, I commented somewhere else why I realized that this actually makes the most sense. Resolving.